Client/Api/Responses/IFormMetadata.cs

```diff
@@ -0,0 +1,16 @@
+namespace Zeebe.Client.Api.Responses;
+
+public interface IFormMetadata
+{
+    /// <returns>the form ID of the deployed form.</returns>
+    string FormId { get; }
+
+    /// <returns>the version of the deployed form.</returns>
+    int Version { get; }
+
+    /// <returns>the key of the deployed form.</returns>
+    long FormKey { get; }
+
+    /// <returns>the name of the deployment resource which contains the form.</returns>
+    string ResourceName { get; }
+}
```

Comment:
❌ I guess we should also add the related tenantId

https://github.com/camunda/zeebe/blob/ab7701b12044584df1660d5509f441ba53546081/clients/java/src/main/java/io/camunda/zeebe/client/api/response/Form.java#L44

Reply:
Added to IFormMetadata